evil-collection.el

```diff
@@ -36,6 +36,7 @@
 (eval-when-compile (require 'subr-x))
 (require 'cl-lib)
 (require 'evil)
+(require 'annalist)
```

Comment:
Is there a performance/startup impact on this require? I know as packages grow, these requires become linearly expensive with that growth.

(An example that springs to mind is lispy which requires a lot of packages!)

Reply:
Annalist only requires cl-lib and subr-x (and I will keep dependencies to a minimum). Requiring org is necessary for displaying keybindings but not when loading annalist or when recording keybindings, so I don't think there should be an issue. Requiring annalist could also be deferred until evil-collection actually makes keybindings, but it's probably unnecessary.